### DISCUSSION TITLE
Code runs on Raspberry

### DISCUSSION THREAD

**Discussion by ap20132022** [raomin/ESPAltherma#232] [Ideas]
can you image that this code also runs on raspberry? 
The reasons for raspberry:
- I have already my volkszaehler (which runs with smart meter) nearby
- in my cellar Wifi works not quite well
- you can use CAN interface with extension
What do you think about this suggest?

**Comment by Tardymo** [comment]
Do you mean the sigle board computer or Pico W?
Communication is done through serial port, so it is possible to port it to any hardware having UART, but it would then be a completely different project.

**Comment by raomin** [comment]
Thanks @Tardymo. That's correct, code would have to be ported to Linux but this would not be very difficult...

**Comment by ap20132022** [comment]
> Do you mean the sigle board computer or Pico W? Communication is done through serial port, so it is possible to port it to any hardware having UART, but it would then be a completely different project.

I mean single board computer. 
Maybe a new project named RaspiTherma or PyTherma?

**Comment by raomin** [comment]
I'll convert this one to a discussion as it's not really an ESPAltherma issue.
